dap_prinz_green_jobs/pipeline/green_measures/industries/sic_mapper/sic_mapper.py

```diff
@@ -440,7 +450,6 @@ def get_sic_codes(self, job_adverts: Union[Dict[str, str], List[Dict[str, str]]]
             )
 
             for job_ad in preprocessed_job_adverts_comp_desc:
-                company_name = job_ad[self.company_name_key]
                 if job_ad["company_description"] != "":
                     company_desc_hash = tc.short_hash(job_ad["company_description"])
```

Comment:
```suggestion
                    company_desc_hash = company_description_to_hash.get(job_ad["company_description"])
```

Reply:
although.. maybe querying a big dictionary like this is more computationally expensive than creating the hash again 🤷‍♀️